File: src/Entity/Commande.php
<?php

namespace App\Entity;

use App\Repository\CommandeRepository;
use Doctrine\Common\Collections\ArrayCollection;
use Doctrine\Common\Collections\Collection;
use Doctrine\ORM\Mapping as ORM;

/**
 * @ORM\Entity(repositoryClass=CommandeRepository::class)
 */
class Commande
{
    /**
     * @ORM\Id
     * @ORM\GeneratedValue
     * @ORM\Column(type="integer")
     */
    private $id;

    /**
     * @ORM\Column(type="string", length=255)
     */
    private $adresse;

    /**
     * @ORM\Column(type="integer")
     */
    private $numtel;

    /**
     * @ORM\Column(type="string", length=255)
     */
    private $nom;

    /**
     * @ORM\Column(type="string", length=255)
     */
    private $prenom;

    /**
     * @ORM\Column(type="string", length=255)
     */
    private $email;

    /**
     * @ORM\Column(type="string", length=255)
     */
    private $etat;

    /**
     * @ORM\ManyToMany(targetEntity=Veloav::class, mappedBy="id_commande")
     */
    private $veloAvs;

    /**
     * @ORM\Column(type="integer")
     */
    private $qunatite;

    /**
     * @ORM\Column(type="datetime")
     */
    private $datecom;


    public function __construct()
    {
        $this->veloAvs = new ArrayCollection();
    }


    public function getId(): ?int
    {
        return $this->id;
    }


    public function getAdresse(): ?string
    {
        return $this->adresse;
    }

    public function setAdresse(string $adresse): self
    {
        $this->adresse = $adresse;

        return $this;
    }

    public function getNumtel(): ?int
    {
        return $this->numtel;
    }

    public function setNumtel(int $numtel): self
    {
        $this->numtel = $numtel;

        return $this;
    }

    public function getNom(): ?string
    {
        return $this->nom;
    }

    public function setNom(string $nom): self
    {
        $this->nom = $nom;

        return $this;
    }

    public function getPrenom(): ?string
    {
        return $this->prenom;
    }

    public function setPrenom(string $prenom): self
    {
        $this->prenom = $prenom;

        return $this;
    }

    public function getEmail(): ?string
    {
        return $this->email;
    }

    public function setEmail(string $email): self
    {
        $this->email = $email;

        return $this;
    }

    public function getEtat(): ?string
    {
        return $this->etat;
    }

    public function setEtat(string $etat): self
    {
        $this->etat = $etat;

        return $this;
    }

    /**
     * @return Collection|VeloAv[]
     */
    public function getVeloAvs(): Collection
    {
        return $this->veloAvs;
    }

    public function addVeloAv(VeloAv $veloAv): self
    {
        if (!$this->veloAvs->contains($veloAv)) {
            $this->veloAvs[] = $veloAv;
            $veloAv->addIdCommande($this);
        }

        return $this;
    }

    public function removeVeloAv(VeloAv $veloAv): self
    {
        if ($this->veloAvs->removeElement($veloAv)) {
            $veloAv->removeIdCommande($this);
        }

        return $this;
    }

    public function getQunatite(): ?int
    {
        return $this->qunatite;
    }

    public function setQunatite(int $qunatite): self
    {
        $this->qunatite = $qunatite;

        return $this;
    }

    public function getDatecom(): ?\DateTimeInterface
    {
        return $this->datecom;
    }

    public function setDatecom(\DateTimeInterface $datecom): self
    {
        $this->datecom = $datecom;

        return $this;
    }

}

File: E-bike_java/src/entities/Commande.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package entities;

import java.sql.Date;

/**
 *
 * @author amine
 */
public class Commande {
 private int id;
 private String adresse;
  private int numtel;
 private String nom;
 private String email;
 private String etat;
 private int quantité;
 private Date datecom;

    public Commande(int id, String adresse, int numtel, String nom, String email, String etat, int quantité, Date datecom) {
        this.id = id;
        this.adresse = adresse;
        this.numtel = numtel;
        this.nom = nom;
        this.email = email;
        this.etat = etat;
        this.quantité = quantité;
        this.datecom = datecom;
    }

    public Commande(String adresse, int numtel, String nom, String email, String etat, int quantité, Date datecom) {
        this.adresse = adresse;
        this.numtel = numtel;
        this.nom = nom;
        this.email = email;
        this.etat = etat;
        this.quantité = quantité;
        this.datecom = datecom;
    }

   

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getAdresse() {
        return adresse;
    }

    public void setAdresse(String adresse) {
        this.adresse = adresse;
    }

    public String getNom() {
        return nom;
    }

    public void setNom(String nom) {
        this.nom = nom;
    }

    public int getNumtel() {
        return numtel;
    }

    public void setNumtel(int numtel) {
        this.numtel = numtel;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    public String getEtat() {
        return etat;
    }

    public void setEtat(String etat) {
        this.etat = etat;
    }

    public int getQuantité() {
        return quantité;
    }

    public void setQuantité(int quantité) {
        this.quantité = quantité;
    }

    public Date getDatecom() {
        return datecom;
    }

    public void setDatecom(Date datecom) {
        this.datecom = datecom;
    }

    @Override
    public String toString() {
        return "Commande{" + "id=" + id + ", adresse=" + adresse + ", numtel=" + numtel + ", nom=" + nom + ", email=" + email + ", etat=" + etat + ", quantit\u00e9=" + quantité + ", datecom=" + datecom + '}';
    }

  
 
   







 
}
